tests/config.spec.coffee

```diff
@@ -4,6 +4,7 @@ timekeeper = require('timekeeper')
 resin = require('resin-sdk-preconfigured')
 errors = require('resin-errors')
 deviceConfig = require('../lib/config')
+_ = require('lodash')
```

Comment:
Yeah, I just use lodash by default as it avoids having to worry about what is available in what version of nodejs/chrome/ie/etc and to get the extra syntactic sugar for a lot of function (eg `_.map(devices, 'id')`) - since lodash is included/used in this module already I'd lean towards just sticking with it, but if you feel strongly I can change it